tests/ballerina-unit-test/src/test/java/org/ballerinalang/test/connectors/sql/SQLActionsTest.java

```diff
@@ -763,6 +763,25 @@ public void testAddToMirrorTableNegative() throws Exception {
         Assert.assertTrue(returns[0].stringValue().contains(errorMessage));
     }
 
+    @Test(groups = CONNECTOR_TEST, description = "Test failure scenario in adding data to mirrored table")
+    public void testAddToMirrorTableInvalidRecord() throws Exception {
```

Comment:
Shall we reword test description and method name use proxy table

Reply:
+1. I'll do this collectively for all places in a separate PR.